.env.sample

```
@@ -206,4 +187,25 @@ AGENT_API_KEY='supersecret-that-too-16chars'
 # VERIFIER_CLIENT_ALIAS=VERIFIER
 # VERIFIER_DOMAIN=https://VERIFIER-domain.com
 # VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_ID=encryptedKeyCloakClientId
-# VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_SECRET=encryptedKeyCloakClientSecret
\ No newline at end of file
+# VERIFIER_KEYCLOAK_MANAGEMENT_CLIENT_SECRET=encryptedKeyCloakClientSecret
+
+# FILE_STORAGE_PROVIDER can be 'aws' or 'minio'
+FILE_STORAGE_PROVIDER=minio
+
+# AWS Configurations
+AWS_ACCOUNT_ID=xxxxx  // Please provide your AWS account Id
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)